netusage@30yavash.com/files/netusage@30yavash.com/settings-schema.json

```diff
@@ -4,41 +4,44 @@
         "description": "Settings for the Network Usage Applet (uses vnstati)"
     },
 
-
     "useExtendedDisplay": {
         "type": "checkbox",
         "default": false,
         "description": "Enable Extended Display Options",
-        "tooltip": "This is new option version 1.0.3"
+        "tooltip": "This is a new option from version 1.0.3"
     },
 
     "extendedDisplay" : {
         "dependency": "useExtendedDisplay",
         "type": "combobox",
         "default": "classic",
         "options": {
-            "Classic - Summary of Data Usage for last Days and Months" : "classic",
-            "Classic Plus - Classic with addition of breakdown by hour for last 24 hours": "classicPlus" ,
+            "Classic - Summary of Traffic for last Days and Months" : "-s",
+            "Classic Plus - Classic with addition of hourly statistics": "-vs" ,
+            "Hourly statistics for last 24 hours" : "-h ",
+            "Daily statistics for last 30 days" : "-d",
+            "Monthly statistics for last 12 months" : "-m",
+            "All time Top ten traffic days" : "-t",
             "Custom Display - requires enabling below": "userDefined"
          },
-        "description" : "Type of Display - default 'classic'",
-        "tooltip": "This feature offers the Classic (original) display with optional of adding an hourly breakdown over the last 24 hours and an experimental option of a user defined string for vnstati when enabled below"
+        "description" : "Type of Display - default Classic",
+        "tooltip": "This feature offers the Classic (original) display with a number of other options including use of a user defined string for vnstati when enabled below"
    },
 
     "useVnstatiCommandString": {
         "dependency": "useExtendedDisplay",
         "type": "checkbox",
         "default": false,
         "description": "Enable Custom Display Option",
-        "tooltip": "Check box"
+        "tooltip": "Beware - no error checking - if you crash the applet try restarting Cinnamon"
```

Comment:
I recommend handling the errors or sanitizing user input instead of instructing people to restart Cinnamon if the desklet crashes.

Reply:
This was somewhat tongue in cheek as it is impossible to know what a program such as vnstati will do with faulty input and I did once upset vnstati with an attempt to use one of the parameters which replaced the vnstati image header with a text string which tripped the the try...catch

There are two choices:
  * Remove the option completely.
  * Make the the message even stronger and recommend backing up first. 

I favour removing it as I have now added all the reasonable display type options and anything else has a poor benefit/risk ratio . The dangers were the reason why there was an extra checkbox to activate it although I found it very useful during testing.

I will push an extra commit to remove the option completely.